primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/GridCssMigration.java

```diff
@@ -45,6 +45,7 @@ protected void initReplaceRegEx() {
 
         // grid
         replaceRegex.put("ui-g-(xl|lg|md|sm)-([0-9]+?)", "p-(xl|lg|md|sm)-([0-9]+?)");
```

Comment:
@melloware shouldn't groups be referenced in the replacement here? I would expect `"p-$1-$2"`.

Reply:
Yes this looks wrong it should be $ like jepsar says

Reply:
In that case we don't have a unit test for that replacement, or it's wrong as well